src/s2wasm.h

```diff
@@ -58,7 +58,6 @@ class S2WasmBuilder {
         startFunction(startFunction),
         globalBase(globalBase),
         nextStatic(globalBase),
-        minInitialMemory(0),
```

Comment:
are you sure this gets written to before it's read? I think that's not the case, and we do need this here (for when there is no stack, no globals, etc.).